vtune/README.md

```diff
@@ -0,0 +1,81 @@
+## VTune analysis
+
+First, the code has been run under VTune on DevCloud "cluster" and we obtained the following picture:  
+
+![](./img/vtune/old/new_core.c_ua1.png) 
+
+As we can see, VTune marks the most CPU consuming task as "Outside any known module", and lots of lines are grayed out and marked "unreliable" when pointed with a mouse. It looks like we do not have proper access to hardware counters on Intel machine, so we went to a local laptop to see more detailed results. Thus the following picture has been obtained:
+
+![](./img/vtune/original_bottomup_uarc.png)
+
+Now we do not have any "Outside any known module" and can see that the function "evolve interior" does, in fact, consume the most resources. Here are the few summaries that VTune gave us:
+
+![](img/vtune/original_memory_summary.png)
+
+We see that the bandwidth we get is rather high, around 28 GB/sec.
```

Comment:
You may refer to a percentage of utilisation instead of pointing out the BW. You can say 28GB/s out of 33GB/s or a percentage, which is more generic.